gen3/bin/kube-setup-audit-service.sh

```diff
@@ -55,12 +64,58 @@ EOM
   gen3 secrets sync 'setup audit-g3auto secrets'
 }
 
+setup_sqs() {
```

Comment:
Do we have any concept of shared setup scripts for our services? `setup_sqs` feels like something other apps could want, and I'd hate to see us have to re-implement this logic every time. If we could do...

```bash
setup_sqs(sender, receiverArray, queueName)
```

...I'd think that'd be a nice abstraction. But maybe better to just put a comment to do that _next_ time we want to add SQS rather than doing the effort now for something that may never be needed.

Reply:
`setup_sqs` calls the new `gen3 sqs` module which is also in this PR 🙂 
`setup_sqs` is specific to the queue we want to create for the audit-service, and the `gen3 sqs` module is generic.
If i move the `fence-sa` policy setup to `kube-setup-fence`, i'll refactor this

Reply:
Sounds good! 👍